web_trackers/README.md

```diff
@@ -9,6 +9,7 @@ It will connects to Snowplow Mini, which is used for development and testing. On
 * [`gov_search_sp_script.js`](./gov_search_sp_script.js) should only be used for Search pages in CMS Lite.
 * [`wordpress_sp_script.js`](./wordpress_sp_script.js) should be used by sites using the standard Wordpress setup. This includes Engage.gov sites and Standalone sites. 
 * [`Snowplow_inline_code.js`](./Snowplow_inline_code.js) can be placed on any Standalone site that uses GDX Analytics services.
```

Comment:
Can you change this to something like:
can be placed on any Standalone site that uses GDX Analytics services that _does not_ have a search function.
to make it clear that they would use _either_ `Snowplow_inline_code.js` or `Snowplow_inline_code_search.js`.

Reply:
I have made this change.